src/platform/forms-system/src/js/review/SubmitController.jsx

```diff
@@ -83,22 +83,21 @@ class SubmitController extends Component {
   };
 
   render() {
-    const { form, formConfig, renderErrorMessage } = this.props;
+    const { form, formConfig } = this.props;
 
     return (
       <SubmitButtons
         formConfig={formConfig}
         onBack={this.goBack}
         onSubmit={this.handleSubmit}
         submission={form.submission}
-        renderErrorMessage={renderErrorMessage}
```

Comment:
`renderErrorMessage` is only used here to pass it down to the child `SubmitButtons`